docs/src/main/sphinx/release/release-363.md

```diff
@@ -0,0 +1,67 @@
+# Release 363 (5 Oct 2021)
+
+## General
+
+* Add `http` event listener implementation which sends JSON serialized events to a remote HTTP endpoint. ({issue}`8821`)
+* Improve performance of queries that group by `bigint` columns. ({issue}`9510`)
+* Reduce query memory usage when spill occurs. ({issue}`9270`, {issue}`9275`)
+* Reduce join memory usage. ({issue}`9327`)
+* Fix potential data loss in query results when clients retry requests to the coordinator. ({issue}`9453`)
+* Fix incorrect result for comparisons between zero-valued decimals. ({issue}`8985`)
+* Fix `SHOW ROLES` failure when there are no roles to display. ({issue}`9319`)
+* Fix `EXPLAIN ANALYZE` to show estimates. ({issue}`9396`)
+* Fix incorrect result for {func}`round` with precision set to 0. ({issue}`9371`) 
+* Respect `deprecated.legacy-catalog-roles=true` configuration property in `SHOW ROLES`, 
+  `SHOW CURRENT ROLES` and `SHOW ROLE GRANTS` statements. ({issue}`9496`)
+
+## Python client
+
+* Fix column type reported in `cursor.description` for `time with time zone` column. ({issue}`9460`)
+
+## BigQuery connector
+
+* Fix failure for queries where predicate on `geography`, `array` or `struct` column is pushed down to BigQuery. ({issue}`9391`)
+
+## Cassandra connector
+
+* Add support for Cassandra tuple type. ({issue}`8570`)
+
+## Elasticsearch connector
+
+* Add support for `scaled_float` type. ({issue}`9358`)
+
+## Hive connector
+
+* Improve query performance when reading Parquet data with predicate on a `decimal` column. ({issue}`9338`)
+* Fix data corruption when performing and insert overwrite operation on S3-backed tables. ({issue}`9234`)
```

Comment:
performing an insert overwrite operation

Reply:
What kind of corruption is this? @losipiuk

Reply:
According to @aczajkowski , it leaves the old data in place and just adds the new data.

Reply:
@aczajkowski, was there a release where the check wasn't present and it resulted in data corruption?

Reply:
@martint This is the commit when check was added:
https://github.com/trinodb/trino/commit/75831d3268e080dcbd256163365ea3ef7a23a383
So seems version **209**

Reply:
Ok, so am I understanding correctly that there was never a data corruption issue, then?